components/__design__/social-logos.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import Image from 'next/image';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+export const SocialLogos: StoryObj = {};
+
+export default {
+  title: 'Design System/Logos',
+  component: () => {
+    return (
+    <div>
```

Comment:
Nit: Please help to fix the indentation so it looks like this:
```js
component: () => {
  return (
    <div>
    ...
    </div>
  );
},
```

And don't forget to fix the indentation in `<Image />` below.